### PR TITLE
Events part 1: events section under community page

### PR DESCRIPTION
**Pull request by darrenvong**
Part of #541 

This is supposed to show the first three events in the community page under the "Events" section, but the API is broken so can't see it for real at the moment. The best alternative is to check Storybook for now.

**Desktop**
Screenshot 2021-07-01 at 02 48 22

**Mobile**
Screenshot 2021-07-01 at 02 51 41

**Frontend checklist**
- [x] Formatted my code with `yarn format && yarn lint --fix`
- [x] There are no warnings from `yarn lint`
- [x] There are no console warnings when running the app
- [x] Added any new components to storybook
- [x] Added tests where relevant
- [x] All tests pass
- [ ] Clicked around my changes running locally and it works
- [x] Checked Desktop, Mobile and Tablet screen sizes

### CODE REVIEW

File: app/frontend/src/features/communities/events/EventCard.tsx [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+    () => getContentSummary(event.content),
+    [event.content]
+  );
+
+  return (
+    <Card
+      className={classNames(className, classes.root)}
+      data-testid={EVENT_CARD_TEST_ID}
+    >
+      <Link to={routeToEvent(event.eventId ?? 0, event.slug ?? "")}>
```
Comment (lucaslcode) [inline app/frontend/src/features/communities/events/EventCard.tsx]: Are these coalescing operators needed?

File: app/frontend/src/features/communities/events/EventCard.tsx [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+        <CardMedia
+          src={event.photoUrl ? event.photoUrl : eventImagePlaceholder}
```
Comment (lucaslcode) [inline app/frontend/src/features/communities/events/EventCard.tsx]: I know I probably wrote this but just use `||` (and not `??`)

File: app/frontend/src/features/communities/events/EventsSection.tsx [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+export default function EventsSection({
+  community,
+}: {
+  community: Community.AsObject;
+}) {
+  const classes = useCommunityPageStyles();
+
+  const { data, error, hasNextPage, isLoading } = useListCommunityEvents({
+    communityId: community.communityId,
+    pageSize: 3,
```
Comment (lucaslcode) [inline app/frontend/src/features/communities/events/EventsSection.tsx:29]: Do 6 - divisible by 3 and 2 (for tablets)

File: app/frontend/src/features/communities/events/EventsSection.tsx [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+  });
+
+  return (
+    <>
+      <SectionTitle icon={<CalendarIcon />} variant="h2">
+        {EVENTS_TITLE}
+      </SectionTitle>
```
Comment (lucaslcode) [inline app/frontend/src/features/communities/events/EventsSection.tsx:36]: This needs a bit of margin. (If it works with the other sections too?)

File: app/frontend/src/features/communities/events/EventCard.tsx [hunk 5]
```diff
@@ -0,0 +1,157 @@
...
+            <Typography
+              variant="caption"
+              component="p"
+              className={classes.subtitle}
+              noWrap
+            >
+              {getByCreator(eventCreator.name)}
+            </Typography>
+          ) : (
+            <Skeleton />
```
Comment (lucaslcode) [inline app/frontend/src/features/communities/events/EventCard.tsx]:
```suggestion
            <Skeleton width={100}/>
```

File: app/frontend/src/features/communities/events/EventCard.tsx [hunk 6]
```diff
@@ -0,0 +1,157 @@
...
+    },
+    "ul&": {
+      listStyle: "none",
+      margin: 0,
+      padding: 0,
+    },
+  },
+  detailsText: {
+    ...theme.typography.body2,
+    color: theme.palette.secondary.main,
```
Comment (lucaslcode) [inline app/frontend/src/features/communities/events/EventCard.tsx:61]: This gives a huge emphasis, I think it should be either plain or primary. But it is like this in the design so I'll leave it to you

File: app/frontend/src/features/communities/constants.ts [hunk 7]
```diff
@@ -11,6 +11,7 @@ export const communityTabBarLabels: Record<CommunityTab, string> = {
 };
 
 export const getByCreator = (name: string) => `By ${name}`;
+export const getAttendeesCount = (count: number) => `${count} attendees`;
```
Comment (lucaslcode) [inline app/frontend/src/features/communities/constants.ts]:
```suggestion
export const getAttendeesCount = (count: number) => count === 1 ? "1 attendee" : `${count} attendees`;
```
Proper i18n "Coming Soon™"

**Review by lucaslcode — CHANGES_REQUESTED** [review]

File: app/frontend/src/features/communities/events/EventCard.tsx (hunk 6 above)
Comment (darrenvong) [reply app/frontend/src/features/communities/events/EventCard.tsx:61]: I kinda like the different colour text for picking out the location easily, but maybe having it bold is too much?

File: app/frontend/src/features/communities/events/EventCard.tsx (hunk 1 above)
Comment (darrenvong) [reply app/frontend/src/features/communities/events/EventCard.tsx]: They are not, copy/pasted/adapted on top of what you had so probably was needed at the time with the old type - oops! I'll delete

File: app/frontend/src/features/communities/events/EventsSection.tsx (hunk 3 above)
Comment (darrenvong) [reply app/frontend/src/features/communities/events/EventsSection.tsx:29]: From the design, don't we only want to show one row of events?

Comment (lucaslcode) [reply app/frontend/src/features/communities/events/EventsSection.tsx:29]: Yeah probably, it looks weird on tablets though.

File: app/frontend/src/features/communities/constants.ts [hunk 8]
```diff
@@ -11,7 +11,8 @@ export const communityTabBarLabels: Record<CommunityTab, string> = {
 };
 
 export const getByCreator = (name: string) => `By ${name}`;
-export const getAttendeesCount = (count: number) => `${count} attendees`;
+export const getAttendeesCount = (count: number) =>
+  `${count} attendee${count === 1 ? "s" : ""}`;
```
Comment (lucaslcode) [inline app/frontend/src/features/communities/constants.ts:12]: This is awful, I love it

**Review by lucaslcode — DISMISSED** [review]
Also the card change as discussed on slack but maybe that's for the next pr

**Comment by darrenvong** [comment]
Had to fix a merge conflict 😢

**Review by lucaslcode — APPROVED** [review]

**Pull request merged**